CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.36.0] - 2018-10-31
+### Updated
+- Updated plotly.js to 1.42.1 [#354](https://github.com/plotly/dash-core-components/pull/354)
+  - https://github.com/plotly/plotly.js/releases/tag/v1.42.0
+  - https://github.com/plotly/plotly.js/releases/tag/v1.42.1
+
```

Comment:
We usually copy over the changelog from plotly.js here, check out other updates that included plotly.js releases below, it should look similar to that.